packages/utils/src/crypto/crypto.browser.ts

```diff
@@ -0,0 +1,13 @@
+import { getRandomValues } from "crypto"
+
+/**
+ * Generates a random bigint value. It uses the Web Cryptography API to securely generate
+ * a random sequence of size bytes and then converts those bytes into a bigint using
+ * big-endian byte order.
+ * @param size The number of bytes.
+ * @returns A randomly generated bigint value.
+ */
+/* eslint-disable import/prefer-default-export */
+export function getRandomValue(size: number): bigint {
```

Comment:
I think the two `getRandomValue` should both return a `Uint8Array` (i.e. a buffer) here. They can convert that buffer to a bigint or whatever with the `conversions` module's functions.

`randomBytes` (on Node) and `getRandomValues` (on browsers) return a buffer, which could be more than 1 value/number, so using the plural sounds more correct here. This is another reason why converting it to bigint here could lead to some issues with its size (bigints don't have the same limit as buffers in size).

Reply:
Thank you so much. Therefore, I will change `bigint` with `Uint8Array` and use the correspondent converters as needed.